### ISSUE TITLE
[bsconfig] validate jsonschema when loading the bsconfig.json files

### ISSUE THREAD

**Issue by bartvandenende-wm** [rokucommunity/brighterscript#1142]
## Summary

[bsconfig] validate `bsconfig.json` against the jsonschema when the json data is loaded.

## Details

It would be great if the `loadConfigFile` utility in https://github.com/rokucommunity/brighterscript/blob/483a15454f1c8751619421ee4c99dbf4fdd073d7/src/util.ts#L168 does not would be able to validate the `bsconfig.json` file against the json-schema defined in https://github.com/rokucommunity/brighterscript/blob/483a15454f1c8751619421ee4c99dbf4fdd073d7/bsconfig.schema.json#L4

**Comment by TwitchBronBron** [comment]
Yeah this would be nice. Find some way to add diagnostics about bad syntax/unknown options.
